src/line_cache.rs

```diff
@@ -31,18 +32,19 @@ fn make_index(mut buf: &[u8]) -> Vec<u32> {
     while buf.len() >= PAGE {
         let mut page = &buf[0..PAGE];
         buf = &buf[PAGE..];
-        // use an i8 accumulator to maximize the effectiveness of vectorization.
-        // do blocks of 128 because we don't want to overflow the i8.  count
+        // use a u8 accumulator to maximize the effectiveness of vectorization.
+        // do blocks of 256 because we don't want to overflow the u8.  count
```

Comment:
My copy of `set.mm` uses Unix newlines, and I haven't actually tested it with Windows newlines (although the new world order means you can pretend `\r` is horizontal whitespace and nearly everything will Just Work)